contextproject-ms3/src/main/java/contextproject/controllers/MenuBarController.java

```diff
@@ -14,4 +26,27 @@
   private MenuItem menuItemImport;
   @FXML
   private MenuItem menuItemExport;
+  @FXML
+  private WindowController controller;
+  
+  
+  @FXML protected void pushButton(ActionEvent event) {
```

Comment:
I think pushButton says too little as a method name. What button is pushed here?